sass/plugin/type/_helpers.scss

```diff
@@ -291,7 +291,8 @@ $_a_FONT-FORMATS: (
   $weight,
   $style,
   $data,
-  $unicode: null
+  $unicode: null,
+  $display: 'auto'
```

Comment:
Since `auto` is the initial value of `font-display`, I don't think there's any reason to include it here as a default. We can use `null` and avoid the explicit setting unless it's explicitly requested.